migration_scripts/to_version_7_0_12/index.js

```diff
@@ -70,10 +68,15 @@ function getNormalizedPhoneNumber(phoneNumber) {
 async function updatePhoneNumbers(table) {
   const batchSize = 1000;
   let offset = 0;
+  let totalUpdatedRows = 0;
 
   try {
     while (true) {
+      let totalRows = await knex.raw(`SELECT COUNT(*) as count FROM ${table} WHERE phone_number is NOT NULL`);
```

Comment:
this query should not be a part of the loop.

Reply:
Thank you so much for catching that. That was really silly. Unfortunately. there was nothing in logs that could point this out. 

I have fixed it now.